### PR TITLE
W3A-17 Endpoint for social user credibility without twitter api

### PR DESCRIPTION
**Pull request by davideluque**

### CODE REVIEW

File: src/calculator/service.ts [hunk 1]
```diff
@@ -175,9 +175,23 @@ async function socialCredibility(userID: string) {
   }
 }
 
+function ScrappedSocialCredibility(followersCount: number, friendsCount: number){
```
Comment (german1608) [inline src/calculator/service.ts]: please, use camelCase, not PascalCase for functions/variables/methods.
PascalCase is ok on classes and interfaces.

Comment (davideluque) [reply src/calculator/service.ts]: > We need tests for this. Check the comment that I left on #11
> Also, I see that the PR name is incorrect for this endpoint. Changing it.

I am pretty sure this is for social credibility, which is different from user credibility. Please confirm if not

Comment (davideluque) [reply src/calculator/service.ts]: I was thinking about writing the tests after the mock data, but now I just recall that this does not need mock data :) Going to do the req changes

**Comment by davideluque** [comment]
I am not sure why my tests are failing, can you help me @german1608

**Comment by german1608** [comment]
> I am not sure why my tests are failing, can you help me @german1608

Yeah sure.

File: src/calculator/service.ts (hunk 1 above)
Comment (german1608) [reply src/calculator/service.ts]: > > We need tests for this. Check the comment that I left on #11
> > Also, I see that the PR name is incorrect for this endpoint. Changing it.
> 
> I am pretty sure this is for social credibility, which is different from user credibility. Please confirm if not
yes, you're right, but at the beginning the title was for tweet credibility. updating it to say that thi is for social

**Comment by davideluque** [comment]
> > I am not sure why my tests are failing, can you help me @german1608
> 
> Yeah sure.

What do you think is the problem?

File: tests/calculus-endpoints/social.test.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        yearJoined: 2000  
+      }
+      it('returns credibility=30.04999998245834 with', () => {
+        return testCredibilityWithOkData({ credibility: 30.04999998245834 }, {
+          followersCount: 1200000,
+          friendsCount: 421,
+          ...params
+        })
+      })
+    })
```
Comment (german1608) [inline tests/calculus-endpoints/social.test.ts]: Mmm I see what are you trying to do in this tests, but I encourage you to use more handable numbers that results in easier credibilities to test with.

For example, you could try this simple combinations:
```
followers = ANY VALUE YOU WANT
following = 0

followers = 0
following = ANY VALUE YOU WANT

followers = ANY VALUE YOU WANT
following = ANY VALUE YOU WANT
```
The ANY VALUE YOU WANT should be values whose final calculations returns a more handable number, not `30.04999998245834`. The test could fail also for floating-point precission and that is sad.

Comment (davideluque) [reply tests/calculus-endpoints/social.test.ts]: Thanks for the info! Also found out that my error is also related to the way I send the parameters, my route is expecting two and I am only sending one!

**Comment by davideluque** [comment]
This is beautiful people

@german1608 @JoseLuisAcv2 @martinezfabiola 

Although it could be better

**Pull request closed without merging**